pkg/certificate/ca.go

```diff
@@ -21,12 +21,13 @@ import (
 
 	kubeoneapi "github.com/kubermatic/kubeone/pkg/apis/kubeone"
 	"github.com/kubermatic/kubeone/pkg/ssh"
-	"github.com/kubermatic/kubeone/pkg/util"
+	"github.com/kubermatic/kubeone/pkg/util/context"
+	"github.com/kubermatic/kubeone/pkg/util/runner"
 )
 
 // DownloadCA grabs CA certs/keys from leader host
-func DownloadCA(ctx *util.Context) error {
-	return ctx.RunTaskOnLeader(func(ctx *util.Context, _ *kubeoneapi.HostConfig, conn ssh.Connection) error {
+func DownloadCA(ctx *context.Context) error {
```

Comment:
I really don't like the name `context.Context`, since it exact clash with https://golang.org/pkg/context/#Context

Reply:
Do you have idea what name should we use instead? In some places I had the `kubeonecontext` import alias, but not sure I'd name the package like that

Reply:
Regarding your edit
> Ideally we should use it in code as kubeone.Context.

I'm against having a package called `kubeone` because I don't really like the idea of having `github.com/kubermatic/kubeone/pkg/util/kubeone` and we already use `kubeone` package for the API, so imports will clash again.